### PR TITLE
Use better placeholder avatars

### PR DESCRIPTION
**Pull request by Ngoakor12**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## Screenshots/videos

image

image

## I solemnly swear that:

- [x] My code follows the style guidelines of this project
- [x] I have merged the develop branch into my branch and fixed any merge conflicts
- [x] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have tested new or existing tests and made sure that they pass

### CODE REVIEW

**Review by kingraphaii — APPROVED** [review]

File: backend/frontend/templatetags/custom_tags.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from django import template
+
+register = template.Library()
+
+
+@register.inclusion_tag("frontend/user/board/partial_user_avatar.html")
+def user_avatar(name, many=False):
```
Comment (sheenarbw) [inline backend/frontend/templatetags/custom_tags.py]: What is `many`? I dont think you are using this argument anywhere. What is it meant to do?

And why does the user avatar take in a name argument? Why not just a user? Why does the calling code need to know that the avtar is derived from the user's name? WHy is that logic scattered everywhere?

Should we even use the user's name? Who knows? If we needed to change that logic we should change it only once. It shouldn't be in every single template. This must be DRY and cohesive.

**Review by sheenarbw — CHANGES_REQUESTED** [review]
What is `many` for?

Also, the code is not cohesive or DRY. Some of the avatar logic is in the calling code, some is in the template tag. 

Remember that we will eventually have profile pics. We should be able to update the code in one place to make it work.

File: backend/frontend/templatetags/custom_tags.py (hunk 1 above)
Comment (Ngoakor12) [reply backend/frontend/templatetags/custom_tags.py]: `Many` was used to conditionally style a group of user avatars. But I moved that to the template.

The avatar logic is now in the template tag.

The user_avatar tag now takes in a user. The user profile is likely to live on the user model, so the only thing that would need updating once we have profile pictures is the user avatar template and the context for the template. We won't need to change all the templates where the user avatar is rendered.

File: backend/frontend/templates/frontend/user/partial_user_avatar.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<div
+  class="bg-slate-400 font-bold {% if many == 'True' %} border-2 border-white {%endif%} text-slate-100 w-[30px] h-[30px] rounded-full flex items-center justify-center flex-nowrap"
```
Comment (kingraphaii) [inline backend/frontend/templates/frontend/user/partial_user_avatar.html]: Is `many` still relevant here

**Review by kingraphaii — CHANGES_REQUESTED** [review]

File: backend/frontend/templates/frontend/user/partial_user_avatar.html (hunk 2 above)
Comment (Ngoakor12) [reply backend/frontend/templates/frontend/user/partial_user_avatar.html]: nope

**Review by kingraphaii — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**